PWGJE/EMCALJetTasks/AliEmcalJetTask.cxx

```diff
@@ -378,6 +378,18 @@ void AliEmcalJetTask::ExecOnce()
   }
 
   fJetsName = AliJetContainer::GenerateJetName(fJetType, fJetAlgo, fRecombScheme, fRadius, GetParticleContainer(0), GetClusterContainer(0), fJetsTag);
+  std::cout << GetName() << ": Name of the jet container: " << fJetsName << std::endl;
+  std::cout << "Use this name in order to connect jet containers in your task to connect to the collection of jets found by this jet finder" << std::endl;
+  if(auto partcont = GetParticleContainer(0)) {
+    std::cout << "Found particle container with name " << partcont->GetName() << std::endl;
+  } else {
+    std::cout << "Not found particle container" << std::endl;
```

Comment:
Trivial, but I would say "The particle container was not found" (same idea for below)

Reply:
OK, but then I would rather have something like "No particle container found for this task". "The particle container was not found" indicates it was expected, but this is not necessarily the case.

Reply:
Sure, "No particle container found for this task" is also good by me. I just found "Not found particle container" to sound a bit strange